Client/src/Validation/validation.js

```diff
@@ -128,10 +128,43 @@ const maintenanceWindowValidation = joi.object({
 	monitors: joi.array().min(1),
 });
 
+const advancedSettingsValidation = joi.object({
+	apiBaseUrl: joi.string().uri({ allowRelative: true }).trim().messages({
+		"string.empty": "API base url is required.",
+		"string.uri": "The URL you provided is not valid.",
+	}),
+	logLevel: joi.string(),
+	systemEmailHost: joi.string().allow(''),
+	systemEmailPort: joi.string().allow(''),
+	systemEmailAddress: joi
+		.string().allow(''),
+	systemEmailPassword: joi.string().allow(''),
+	jwtTTL: joi
+		.string()
+		.trim()
+		.messages({
+			"string.empty": "JWT TTL is required."			
+		}),
+	dbType: joi.string().trim().messages({
+		"string.empty": "DB type is required",
+	}),
+	redisHost: joi.string().trim().messages({
+		"string.empty": "Redis host is required",
+	}),
+	redisPort: joi.string().allow('').custom((value, helpers) => {
+		if(value && isNaN(parseInt(value))){
```

Comment:
`isNan` coerces its operand to into a number, so it doesn't always behave the way you want it to.

[Recommended approach](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isNaN#description) is to use `Number.isNan()` instead.

Reply:
fixed in https://github.com/bluewave-labs/bluewave-uptime/pull/986/commits/8ec2f2f16b916eb15cd5d092da768e29b55e6da2, thanks!

Reply:
@jennifer-gan looks like `isNan` is still present in the latest commit?  Maybe it didn't get added?